src/screens/Signup/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import SignupForm from '../../components/SignupForm';
+
+function Signup() {
+  return <SignupForm />;
+}
+
+export default Signup;
```

Comment:
again, this component is totally unnecessary

Reply:
how should i do it?